src/abax-client.ts

```diff
@@ -88,7 +88,12 @@ export class AbaxClient {
       .args<{ input: ListVehiclesInput }>()
       .method('get')
       .path('v1/vehicles')
-      .query(({ input }) => makeQuery(input))
+      .query(({ input }) =>
+        makeSearchParams({
+          page: input.query?.page,
+          page_size: input.query?.pageSize,
+        }),
```

Comment:
We should ditch the QueryEnvelope stuff, so that these fields would sit directly on the input object. The user of the SDK shouldn't have to worry about what goes in the query and what goes in the body. They just provide the inputs.

Reply:
It might make sense to do this in a separate PR, though

Reply:
I was thinking we could do the QueryEnvelope as its own PR, as that caused some cascading errors when refactoring

Reply:
I agree it should go, I was just thinking of splitting the refactoring into a couple of smaller PRs to solve one problem at a time